srcpkgs/apparmor/template

```diff
@@ -47,7 +47,9 @@ pre_build() {
 
 post_build() {
 	cd ${wrksrc}
-	sed -i 's#$(DESTDIR)/sbin#$(DESTDIR)/usr/bin#g' parser/Makefile
+	sed -i 's#SBINDIR=${DESTDIR}/usr/sbin#SBINDIR=${DESTDIR}/usr/bin#' binutils/Makefile
+	sed -i 's#SBINDIR=${DESTDIR}/usr/sbin#SBINDIR=${DESTDIR}/usr/bin#' utils/Makefile
+	sed -i 's#/sbin#/usr/bin#g' parser/Makefile
```

Comment:
probably want to use `vsed` here to check sed lines that no longer make changes (s